### PR TITLE
Feature/standalone helm installer

### PR DESCRIPTION
**Pull request by harishp8889**
# Description
Updated helm installer by removing shell scripting and performing validations through helm itself

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/355  |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Tested on Kubernetes cluster

### CODE REVIEW

File: dell-csi-helm-installer/csi-install.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ PROG="${0}"
 NODE_VERIFY=1
 VERIFY=1
 MODE="install"
-DEFAULT_DRIVER_VERSION="2.0.0"
+DEFAULT_DRIVER_VERSION="v2.0.0"
```
Comment (spriya-m) [inline dell-csi-helm-installer/csi-install.sh:18]: Driver version to be changed to 2.3.0?

File: helm/csi-powerstore/templates/controller.yaml [hunk 2]
```diff
@@ -296,7 +296,7 @@ spec:
             - name: socket-dir
               mountPath: /var/run/csi
         - name: driver
-          image: {{ required "Must provide the PowerStore driver container image." .Values.images.driver }}
+          image: {{ required "Must provide the Powerstore driver image repository." .Values.images.driverRepository }}/{{ .Chart.Name }}:{{ .Values.version }}
```
Comment (spriya-m) [inline helm/csi-powerstore/templates/controller.yaml:299]: Powerstore => PowerStore
same comment for line 94 also.

**Comment by spriya-m** [comment]
Please update the github issue.

File: helm/csi-powerstore/templates/controller.yaml (hunk 2 above)
Comment (harishp8889) [reply helm/csi-powerstore/templates/controller.yaml]: Fixed

File: dell-csi-helm-installer/csi-install.sh (hunk 1 above)
Comment (harishp8889) [reply dell-csi-helm-installer/csi-install.sh:18]: Nope, this should be `v2.0.0` only after this new helm change, Same format is maintained across all the drivers

**Comment by harishp8889** [comment]
> Please update the github issue.

done

File: dell-csi-helm-installer/csi-install.sh (hunk 1 above)
Comment (spriya-m) [reply dell-csi-helm-installer/csi-install.sh:18]: ok

**Review by spriya-m — APPROVED** [review]

**Review by harshitap26 — APPROVED** [review]

**Pull request merged**